lib/src/main/java/org/team100/lib/motor/drive/Falcon6DriveMotor.java

```diff
@@ -334,28 +321,28 @@ public void reset() {
     private static double frictionFF(double currentMotorRev_S, double desiredMotorRev_S) {
         double direction = Math.signum(desiredMotorRev_S);
         if (currentMotorRev_S < staticFrictionSpeedLimitRev_S) {
-            return staticFrictionFFVolts * direction / saturationVoltage;
+            return staticFrictionFFVolts * direction;
```

Comment:
still looks like there's confusion about the unit here.  is it duty cycle or volts?

Reply:
Changed to VelocityTorqueCurrentFOC